HW04/ekaterina_balabanovich/lib/work_state/new.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module WorkState
+  class New < Base
+    def process(members, homework)
+      WorksManager.instance.notify(members: members, task: homework, status: :new)
+      if members.is_a? Array
+        members.each { |member| member.add_homework(homework) }
+      else
+        members.add_homework(homework)
+      end
```

Comment:
This is a bad approach. If you call something `members` then we should always receive an array, but I don't know how difficult will be to wrap all these single cases into an array 
```suggestion
      members.each { |member| member.add_homework(homework) }
```